Storage/Rust/RustFirefoxSuggest.swift

```diff
@@ -0,0 +1,43 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import MozillaAppServices
+
+/// An actor that wraps the synchronous Rust `SuggestStore` binding to execute
+/// blocking operations on the default global concurrent executor.
+public actor RustFirefoxSuggest {
+    private let store: SuggestStoreProtocol?
+
+    public init(databasePath: String, remoteSettingsConfig: RemoteSettingsConfig? = nil) {
+        store = try? SuggestStore(path: databasePath, settingsConfig: remoteSettingsConfig)
+    }
+
+    /// Downloads and stores new Firefox Suggest suggestions.
+    public func ingest() async throws {
+        // Ensure that the Rust networking stack has been initialized before
+        // downloading new suggestions. This is safe to call multiple times.
+        Viaduct.shared.useReqwestBackend()
```

Comment:
`useReqwestBackend` is this a typo?

Reply:
[Nope!](https://github.com/mozilla/application-services/blob/235e3d8b8674a66cf09788d821c85f3f293754d6/components/viaduct/ios/Viaduct.swift#L22-L26)

["Reqwest"](https://docs.rs/reqwest/latest/reqwest/) is the Rust networking library that we use to make HTTP requests from Rust code in Firefox for iOS 😊 [`RustFirefoxAccounts`](https://github.com/mozilla-mobile/firefox-ios/blob/e44e65bfcbb015264db5b64ea01863d4328a8be8/RustFxA/RustFirefoxAccounts.swift#L151-L153) makes the same call, but it's safe to do multiple times, and will only initialize the backend once on the Rust side.